app/controllers/admin/routes.py

```diff
@@ -155,6 +154,26 @@ def addRecurringEvents():
     recurringEvents = calculateRecurringEventFrequency(preprocessEventData(request.form.copy()))
     return json.dumps(recurringEvents, default=str)
 
+@admin_bp.route('/courseProposals', methods=['GET'])
+def createTable():
+    courseDict = getProposalData(g.current_user)
+    try:
+        return render_template("/admin/courseProposals.html",
+                                instructor = g.current_user,
+                                courseDict = courseDict)
+    except Exception as e:
+        print('Error while creating table: ', e)
+        return "", 500
```

Comment:
There's no reason for this try/except. Any exception will already trigger a 500 error and print the message by default. You should only use a blanket try/except like this if you are needing to catch a specific exception, if you want to suppress or sanitize an error message, or if you want to continue through an error

Reply:
try/except removed!